sympy/tensor/index_methods.py

```diff
@@ -24,6 +23,7 @@ class IndexConformanceException(Exception):
     pass
 
 
+@deprecated(useinstead='EinsteinSum', issue=9284)
 def _remove_repeated(inds):
```

Comment:
That's not a good plan actually. Depending on how the deprecation works, one should either:

1.) Update the doctest to not throw a warning (usually for deprecations of kwargs, interfaces, etc...)
2.) Add a `# doctest +SKIP`next to the lines

Having deprecation warnings appear in the output is unhelpful and silly - doctests exist purely to ensure that the docs don't fall out of date, not as tests for functionality. That's why we have normal tests.